resources/views/checkout/steps/success.blade.php

```diff
@@ -9,7 +9,6 @@
         <x-rapidez-ct::sections>
             @include('rapidez-ct::checkout.partials.sections.success.order-info')
             @include('rapidez-ct::checkout.partials.sections.success.products')
-            @include('rapidez-ct::components.newsletter')
```

Comment:
But this is designed to be on the success page right?

Reply:
I don't think it ever was. Besides the fact that it wouldn't work as it stands right now, I'm also confused about the UX of having a checkbox like this in the checkout success. Something worth discussing though :)